src/index.html

```diff
@@ -1,17 +1,178 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"
+    >
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          How old are You?
+          <input
+            name="age"
+            id="age"
+            type="number"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-label">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            id="birth"
+          >
+        </label>
+
+        <label class="form-label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+
+        <label class="form-label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="20"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-label">
```

Comment:
now there is a bug: click on `Do you love cats` select first radio input.
replace on div here and wrap each input in label.
```suggestion
        <div class="form-label">
```